filter/big_filter/filtertobig/filter_to_big.go

```diff
@@ -0,0 +1,8 @@
+package filtertobig
```

Comment:
`filter_to_big`

But it's also fine to have this in the `big_filter` namespace.